apache-http-client/src/test/java/com/joyent/http/signature/apache/httpclient/HttpSignatureRequestInterceptorTest.java

```diff
@@ -66,12 +66,12 @@ public void signBenchmark() throws IOException, HttpException {
                 new BasicHeader("User-Agent", "Java-Manta-SDK/3.0.0-SNAPSHOT (Java/1.8.0_76/Oracle Corporation)"),
                 new BasicHeader("Accept-Encoding", "gzip,deflate"),
         };
-        HttpRequest request = new HttpPut("https://us-east.manta.joyent.com:443");
-        request.setHeaders(headers);
 
         HttpContext context = new BasicHttpContext();
 
-        for (int i = 0; i < 5; i++) {
+        for (int i = 0; i < 500; i++) {
+            HttpRequest request = new HttpPut("https://us-east.manta.joyent.com:443");
+            request.setHeaders(headers);
```

Comment:
These lines moved because the Date header value is not regenerated per iteration without regenerating the request. By moving it into the loop, we can see the effects of header caching.